DataTool/ToolLogic/Extract/Debug/Unused/ExtractDebugPackageFinder.cs

```diff
@@ -6,7 +6,7 @@
 using static DataTool.Program;
 
 namespace DataTool.ToolLogic.Extract.Debug {
-    [Tool("extract-debug-packagefinder", Description = "Test package loading (debug)", CustomFlags = typeof(ExtractFlags), IsSensitive = true)]
+    [Tool("extract-debug-packagefinder", Description = "Test package loading (debug)",, CustomFlags = typeof(ExtractFlags), IsSensitive = true)]
```

Comment:
ah yes

Reply:
come on, these tools broken anyways if you uncomment them

Reply:
Well they are now.